utilities/sanitizeTrend.js

```diff
@@ -1,5 +1,13 @@
-const sanitizeTrend = (rawTimeline) => {
-  return JSON.parse(rawTimeline).default.timelineData.map((point) => {
+module.exports = (rawTimeline) => {
+  let parsedTimeline = {};
+
+  try {
+    parsedTimeline = JSON.parse(rawTimeline);
+  } catch (error) {
+    throw error;
```

Comment:
I was thinking that *not* assuming that it's a string is more resilient as it flags anything that is not an object ie, not what we expected.